src/Router.js

```diff
@@ -44,6 +45,17 @@ class RouterComponent extends Component {
                         rightIcon="search"
                         title="ALERTS & NOTIFICATIONS"
                     />
+                    <Scene
+                        key="filters"
+                        sceneStyle={{paddingTop: STATUSBAR_HEIGHT}}
+                        component={FiltersShip}
+                        navBar={CustomNavBar}
+                        icons={true}
+                        leftIcon="keyboard_arrow_down"  // TODO - arrow-left
```

Comment:
I have already added arrow-left on GlobalMap component, when I merge it to dev you can use it too. :)